### DISCUSSION TITLE
the chinese version is not same as english version

### DISCUSSION THREAD

**Discussion by xiaohu-zhang** [google/comprehensive-rust#2373] [General]
pr:https://github.com/google/comprehensive-rust/pull/2372 is available to see in english version.but why in chinese version 
https://google.github.io/comprehensive-rust/zh-CN/exercises/concurrency/solutions-afternoon.html that is still the old version,there is no changing.

**Comment by mgeisler** [comment]
Hi @xiaohu-zhang,

Yes, you are correct: each translation is frozen in time. Once in a while (when someone finds time), the translators follow the steps listed here: [Refreshing an Existing Translation](https://github.com/google/comprehensive-rust/blob/main/TRANSLATIONS.md#refreshing-an-existing-translation).

You can do this too! Someone will review the PR and after it is merged, you can start translating the new strings added since the last refresh.

**Reply by xiaohu-zhang** [reply]
thank you very much.I will find time do it
